include/radio.hpp

```diff
@@ -64,15 +64,26 @@ namespace bzzz
          */
         int m_encodedSwitchesData;
 
+        /**
+         * Boolean to enable sending IMU data to Pi whenever ESP receives radio data
+         * if True: replies with IMU data;  
+        */
+       bool m_replyWithIMUData;
+
     public:
-        Radio();
+        Radio(bool replyWithIMUData = false);
```

Comment:
This is undocumented. Moreover, I don't understand how this works.

Reply:
Renamed `replyWithIMUData` to `replyWithFlightData`. Documentation added.